nav2_behavior_tree/include/nav2_behavior_tree/bt_service_node.hpp

```diff
@@ -105,6 +103,14 @@ class BtServiceNode : public BT::CoroActionNode
   {
   }
 
+  // An opportunity to do something after
+  // a timeout waiting for a result that hasn't been received yet
+  virtual void on_service_timeout()
+  {
+    RCLCPP_WARN(node_->get_logger(), "\"%s\" service timed out with tolerance: %dms",
```

Comment:
This virtual should be empty. If someone overrides it to use the function, we'd still like this warning to appear, see commment above

Reply:
Right, my mistake.